client/state/signup/optional-dependencies/selectors.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import get from 'lodash/get';
```

Comment:
Here we can use the single member import syntax, which is the recommended one recently:

```
import { get } from 'lodash';
```